pyproject.toml

```
@@ -30,6 +30,7 @@ jinja2 = ">=2.0"
 xonsh = ">=0.14.0"
 python-dotenv = ">=0.21.1"
 colorama = ">=0.4.6"
+typer = "^0.9.0"
```

```suggestion
typer = ">=0.9.0"
```

IIRC the "^" operator will be removed from poetry in versión 2, so probably is better to not use that anymore